File: java/oop/CollegeApp/src/com/techlabs/college/Person.java
package com.techlabs.college;

public class Person {
	private int id;
	private String address;
	private String dob;
	private String name;
	private Branch branch;

	
	public Person(String name,String address,String dob,int id,Branch branch){
		this.name=name;
		this.address=address;
		this.dob=dob;
		this.id=id;
		this.branch=branch;
	}
	public int getId() {
		return this.id;
	}

	public String getAddress() {
		return this.address;
	}
	
	public String getDob(){
		return this.dob;
	}
	
	public String getName(){
		return this.name;
	}
	
	public Branch getBranch(){
		return this.branch;
	}
}

File: dotNet/csharp/CollegeApp/CollegeApp/Person.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CollegeApp
{
    class Person
    {
        private string _name;
        private int _id;
        private string _address;
        private string _dob;
        private Branch branch;

        public Person(string name,string address,string dob,int id,Branch branch)
        {
            this._name = name;
            this._address = address;
            this._dob = dob;
            this._id = id;
            this.branch = branch;
        }

        public string Name
        {
            get
            {
                return _name;
            }
        }

        public string Address
        {
            get
            {
                return _address;
            }
        }

        public string Dob
        {
            get
            {
                return _dob;
            }
        }

        public int Id
        {
            get
            {
                return _id;
            }
        }

        public string Branch
        {
            get
            {
                return this.branch.ToString();
            }
        }
    }
}
